docs/metrics.md

```diff
@@ -144,8 +148,10 @@ Metrics below are captured within load balancer
 
 Metrics below are for invoker state as recorded within load balancer monitoring.
 
-* `openwhisk.counter.loadbalancer_invokerOffline_count` - Records the count of invokers considered offline based on health pings.
-* `openwhisk.counter.loadbalancer_invokerUnhealthy_count` - Records the count of invokers considered unhealthy based on health pings.
+* `openwhisk.counter.loadbalancer_totalHealthyInvoker<invokerType>_count` - Records the count of managed invokers considered healthy based on health pings. `invokerType` defines whether it is a managed or a blackbox action.
```

Comment:
Are these values supposed to be recorded as counter or [guages][1]

> Counters: Track the number of times certain event happened. It can only be increased. Useful for counting errors, cache misses, etc.
> Gagues: Track a single value that can be increased, decreased or explicitly set. The common use case for gauges are slowly moving values like disk usage, number of loaded classes or configuration settings.

Per current impl they seem to be recorded as Histogram

https://github.com/apache/incubator-openwhisk/blob/28ae83cfb1bd07f55b12dc91b16f0e113dbfee2f/core/controller/src/main/scala/org/apache/openwhisk/core/loadBalancer/ShardingContainerPoolBalancer.scala#L186-L209

[1]: https://kamon.io/documentation/1.x/core/basics/metrics/#gauges

Reply:
@chetanmeh Good catch. They are recorded Histogram indeed. Will do the change.

Reply:
@SugandhaAgrawal What I meant was that the kind of metrics we record here in code is supposed to be `gauge` and histogram may not be the right way to capture it. So we may need to change the implementation as well

Reply:
@chetanmeh I checked in the code if we have ever used **gauges** before. I saw 3 examples where indeed this could have been gauges but they are implemented as histogram instead. 1) DB Batch size for CouchDB, 2) Client retries in Akka client and 3) Kafka queue size. I would propose to keep Histogram for the above mentioned metrics. Gauge implementation have to be done in code, and once this is done, I propose changing all the incorrect metrics.

Reply:
Makes sense. Can you create an issue to track this change? As with current histogram based metric its hard to use these value for any monitoring purpose

Reply:
Sure thing. @chetanmeh can you please merge this one in the meanwhile?